go.mod

```diff
@@ -171,4 +171,5 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.30.2
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.30.2
 	k8s.io/sample-controller => k8s.io/sample-controller v0.30.2
+	volcano.sh/apis => github.com/volcano-sh/apis v1.10.0-alpha.0.0.20240709084748-78d912ce096c
```

Comment:
I couldn't do it without adding this line(

```bash
go get github.com/volcano-sh/apis@master
go: github.com/volcano-sh/apis@master (v1.10.0-alpha.0.0.20240709084748-78d912ce096c) requires github.com/volcano-sh/apis@v1.10.0-alpha.0.0.20240709084748-78d912ce096c: parsing go.mod:
        module declares its path as: volcano.sh/apis
                but was required as: github.com/volcano-sh/apis
```

Perhaps this is due to the fact that the changes have been included in the master, but have not yet been included in any tag.

As far as I understand, I still have to wait for the next tag to appear in [volcano-sh/apis](https://github.com/volcano-sh/apis) and then make changes again?

Reply:
Thanks, it worked